internal/data/disbursements.go

```diff
@@ -332,6 +336,7 @@ func (d *DisbursementModel) GetAll(ctx context.Context, sqlExec db.SQLExecuter,
 			d.created_at,
 			d.updated_at,
 			d.verification_field,
+			d.sms_registration_message_template,
```

Comment:
Given that `sms_registration_message_template` field is null and is not set as a pointer on `disbursement`, it will break all these methods. 

We have a few options: 
1. Coalesce when fetching
2. Use a pointer on the struct 
3. Make the database field not nullable and default to empty string.
 

![Screenshot 2024-01-15 at 5 10 33 PM](https://github.com/stellar/stellar-disbursement-platform-backend/assets/4129193/5725729b-34cc-405d-a997-95622c8d2140)